ebuild-writing/functions/src_test/text.xml

```diff
@@ -96,12 +95,6 @@ using <c>sed</c> or skipping a particular <c>make</c> target is
 sufficient.
 </p>
 
-<note>
-<c>emake</c> should not be used for <c>src_test</c> <d/> trying to
-parallelise tests unless the <c>Makefile</c> was specifically designed
-for this can cause all sorts of strange problems.
-</note>
-
```

Comment:
I like the direction of saying `use emake -j1` instead of `do not use emake`. I wonder if we should update this sentence and make it compliant with the example above or simple remove it. Not sure personally.